README.md

```diff
@@ -98,21 +160,27 @@ func AddUser(args interface{}) error {
 
 The manual changes you need to do are:
 
-```go
-//autometrics:inst
-func AddUser(args interface{}) (err error) { // Name the error return value; this is an optional but recommended change
+```patch
++//autometrics:inst
+-func AddUser(args interface{}) error {
++func AddUser(args interface{}) (err error) {
```

Comment:
go 1.18 renamed `interface{}` to `any`, we should prefer that instead of relying on the alias: https://stackoverflow.com/a/71964388/11494565